### ISSUE TITLE
After Creating Server, Serve Not Showed in WHMCS

### ISSUE THREAD

**Issue by shahidmalla1337sm** [vultr/whmcs-vultr#111]
1. Login to to client area
2. click on my service 
3. select location after that server created but now in whmcs
4. See error in screenshot when server already created in server but not show in whmcs

https://prnt.sc/17c09zh    https://prnt.sc/17c0gfx

**Comment by Anubianhost** [comment]
I am now running in to the same issue, It appears the update to WHCMS 8.2 broke the API again.

**Comment by shahidmalla1337sm** [comment]
i am trying to fix the issue, please let me know if you found any solution

**Comment by shahidmalla1337sm** [comment]
https://www.vultr.com/api/#operation/list-instances

**Comment by meramsey** [comment]
check my fork and release 
https://github.com/meramsey/whmcs-vultr/releases/tag/2.0.5

**Comment by Anubianhost** [comment]
Thanks so much @meramsey  Tested and this is working again with Version: 8.2.0 of WHMCS again!

**Comment by Anubianhost** [comment]
@meramsey Not sure what happened. My test went through and the Virtual Machine ID loaded in WHMCS but just sold a New VPS today and the user created it and WHCS never got it and it allowed the user to try again.

**Comment by meramsey** [comment]
Yeah i think ive seen that happen once due to it letting clients click create multiple times if they do that vs being patient and waiting for it to complete. Was an issue before too afaik.  Can report a bug on my fork if you would like not sure how much time ill have to look into it just yet. When this happens i usually delete the duplicates and just enter the first uuid/subid and save in the product to link it back. 

In reality i feel like this module could/should probably be rewritten from scratch at for some sections as there are some really weird issues at times that shouldn't really be possible.

**Comment by Anubianhost** [comment]
Yeah I have to manually go in and update the VM ID when this happens but that was really the only issue I was having. 

We just updated to WHMCS 8.2.1.  I'll try a couple tests runs and see if it continues to do it again.

**Comment by HackersThirdEyeofficial** [comment]
still not fixed I update a lot of things but still not reliable even after creating server-client don't see manage server

**Comment by Anubianhost** [comment]
Same, I had a new order come in and it worked but one other one didn't seems random.

**Comment by meramsey** [comment]
Advise looking into this maintained product which is what I switched too.
https://www.modulesgarden.com/products/whmcs/vultr-vps

 This made it finally possible to transition the existing clients in place by adding a new customfield to all existing Vultr Products "instanceId|Instance ID" text admin only. I then went and populated all existing orders with their V2 API instance id which you can get from visiting the server in the my.vultr.com dashboard and noting the string from url or via the V1/V2 list servers/instances API "v2_id": field. I then was able to just switch the products Module from Vultr to Vultr VPS and all the existing server's connections remain working.
